lib/puppet/indirector/hiera.rb

```diff
@@ -69,6 +69,8 @@ def convert_merge(merge)
     end
   end
 
+  public
```

Comment:
Also i'm leaning towards having class methods in a `class << self` at top because it will help us keep the class methods grouped.

Reply:
I also prefer the style you're talking about, but that kind of refactor is more than I wanted to do. If we did do something like that, then I'd only want to do it in `master` to avoid massive merge conflicts.